src/test/java/org/opentripplanner/middleware/TestUtils.java

```diff
@@ -115,13 +129,24 @@ public static boolean getBooleanEnvVar(String var) {
             if (otpUserFromDB != null && otpUserFromDB.applicationId != null) {
                 return headers;
             }
+            scope = OtpUser.SCOPE;
         }
 
         // Otherwise, get a valid oauth token for the user
-        headers.put("Authorization", "Bearer " + Auth0Users.getAuth0AccessToken(requestingUser.email, TEMP_AUTH0_USER_PASSWORD));
+        headers.put("Authorization", "Bearer " + getMockAuth0AccessToken(requestingUser.email, scope));
         return headers;
     }
 
+    private static String getMockAuth0AccessToken(String username, String scope) {
```

Comment:
Method name feels a little off -- We are getting a real Auth0 token here, right? Maybe just `getTestAuth0AccessToken`? Also, could use some javadoc.

Reply:
Updated.